lib/index.js

```diff
@@ -23,60 +23,58 @@ exports.assertCompatible = (A, B, msg) => {
     Hoek.assert(isExtension && nameMatches && tablenameMatches, msg || 'Models are incompatible.  One model must extend the other, they must have the same name, and share the same tableName.');
 };
 
-exports.register = (server, options, next) => {
-
-    Joi.assert(options, Schema.plugin, 'Bad plugin options passed to schwifty.');
+exports.plugin = {
```

Comment:
How was this `plugin` property introduced?  I see the reason partly, but I'm surprised it works as used in the tests!

Reply:
yo! I saw it on the release notes, plugin section

Realizing now that the release notes do a much clearer job of explaining this than the API docs I guess b/c the API docs cover just the object structure, not clearly showing an exports example? i dunno